### DISCUSSION TITLE
Graphic previews in the file

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#556] [Q&A]
Okay, this is decidedly sub-optimal:

![image](https://github.com/user-attachments/assets/e77c2290-26ea-4e75-887c-f862e83e8fb3)

Why is this displayed this way?

These images are 1500 x 1000 pixels.

**Comment by doobes** [comment]
Ping

**Comment by jeremypw** [comment]
I have converted this to an issue as it looks like there has been a regression in the thumbnailing.  Files used to show large thumbnails in IconView above a certain zoom level.

**Comment by doobes** [comment]
Thank you!

**Comment by jeremypw** [comment]
See https://github.com/elementary/files/issues/2455 for associated issue.

**Comment by doobes** [comment]
Well it's marked as fixed, but the behavior remains.

**Comment by jeremypw** [comment]
https://github.com/elementary/files/issues/2455 is still "Open" i.e. no fix has been merged into the main branch.  There is a linked PR that does fix the issue.  When that is merged then anyone running the unstable (daily) version of elementary will see the fix.  Otherwise you will have to wait for the next stable release of Files for 7.x
